app/(header-default)/documents/[slug]/_components/summary-documents.tsx

```diff
@@ -0,0 +1,25 @@
+import { EScope, hasRights } from '#models/user/rights';
+import { ISession } from '#models/user/session';
+
+export const SummaryDocuments = ({ session }: { session: ISession | null }) => (
+  <>
```

Comment:
D'un point de vue accessibilité, je pense qu'il serait mieux que le menu soit dans un nav : 
```suggestion
 <nav role="navigation" aria-labelledby="document-summary-title">
   <h3 id="document-summary-title">     Documents disponibles pour cette structure :</h3>
```

On peut remplacer le `h3` par un `strong` au besoin.